agreement/service_test.go

```diff
@@ -974,10 +984,47 @@ func TestAgreementSynchronous5_50(t *testing.T) {
 	simulateAgreement(t, 5, 50, disabled)
 }
 
+func TestAgreementSynchronousFuture1(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping agreement integration test")
+	}
```

Comment:
Let's take this block out and always run the one-node test (they're relatively fast).